tests/profiling/collector/test_traceback.py

```diff
@@ -14,11 +14,9 @@ def test_check_traceback_to_frames():
         exc_type, exc_value, traceback = sys.exc_info()
     frames, nframes = _traceback.traceback_to_frames(traceback, 10)
     assert nframes == 2
+
+    this_file = __file__.replace(".pyc", ".py")
```

Comment:
why this change? I am not sure how it is related

Reply:
Python 2.7 started failing on these tests because `__file__` had a `.pyc` extension. Not sure why this is happening, but I suspect it's the import mechanism from the `ModuleWatchdog`